test/assert.js

```diff
@@ -747,6 +747,22 @@ describe('assert', function () {
       assert.doesNotHaveAnyKeys(testMap, [ 'thisDoesNotExist', 'thisToo', {iDoNot: 'exist'} ]);
       assert.doesNotHaveAllKeys(testMap, [ aKey, {iDoNot: 'exist'} ]);
 
+      // Tests for the deep variations of the keys assertion
```

Comment:
I'm thinking a test should be added to the non-deep variety of assertion above demonstrating that it uses strict equality instead of deep. For example:
```js
assert.doesNotHaveAnyKeys(testMap, {thisIs: 'anExampleObject'});
```